src/cockpit/389-console/src/lib/server/accessLog.jsx

```diff
@@ -361,358 +421,353 @@ export class ServerAccessLog extends React.Component {
-                                        </Col>
-                                    </Row>
-                                    <Button
-                                        isDisabled={this.state.saveRotationDisabled}
-                                        variant="primary"
-                                        className="ds-margin-top-med"
-                                        onClick={() => {
-                                            this.saveConfig("rotation");
+                                    />
+                                </GridItem>
+                            </Grid>
+                            <Grid title="The maximum size of each log file in megabytes (nsslapd-accesslog-maxlogsize).">
+                                <GridItem className="ds-label" span={3}>
+                                    Maximum Log Size (in MB)
+                                </GridItem>
+                                <GridItem span={3}>
+                                    <TextInput
+                                        value={this.state['nsslapd-accesslog-maxlogsize']}
+                                        type="number"
+                                        id="nsslapd-accesslog-maxlogsize"
+                                        aria-describedby="horizontal-form-name-helper"
+                                        name="server-accesslog-maxlogsize"
+                                        onChange={(str, e) => {
+                                            this.handleChange(e, "rotation");
+                                        }}
+                                    />
+                                </GridItem>
+                            </Grid>
+                            <hr />
+                            <Grid title="Rotate the log based this number of time units (nsslapd-accesslog-logrotationtime).">
+                                <GridItem className="ds-label" span={3}>
+                                    Create New Log Every ...
+                                </GridItem>
+                                <GridItem span={1}>
+                                    <TextInput
+                                        value={this.state['nsslapd-accesslog-logrotationtime']}
+                                        type="number"
+                                        id="nsslapd-accesslog-logrotationtime"
+                                        aria-describedby="horizontal-form-name-helper"
+                                        name="server-accesslog-logrotationtime"
+                                        onChange={(str, e) => {
+                                            this.handleChange(e, "rotation");
+                                        }}
+                                    />
+                                </GridItem>
+                                <GridItem span={1} className="ds-left-margin">
+                                    <FormSelect
```

Comment:
I wonder if we can adjust the field size here. I am using Chrome with 125% zoom (as it's comfortable for my eyes). And the select doesn't show full 'day' word - 'd...'.
We have plenty of space on the right size so we probably can adjust it a bit:
![image](https://user-images.githubusercontent.com/5634892/124972972-4291d080-e02b-11eb-8ead-03d0a10ac7fa.png)

Reply:
Fixed